tests/helpers/tooling_contracts/makefile.py

```
@@ -0,0 +1,132 @@
+"""Assert rendered Makefile contracts for generated Rust projects."""
+
+from __future__ import annotations
+
+import re
+import tempfile
+from pathlib import Path
+
+import make_parser
+
+
+def _assert_makefile_contracts(makefile: str, flavour: str) -> None:
```

Ruff (0.15.15)

[warning] Unused function argument: `flavour`

(ARG001)